examples/tc_port_whitelist/src/bpf/tc.bpf.c

```diff
@@ -0,0 +1,297 @@
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_endian.h>
+
+//#include <linux/if_ether.h>
+#define ETH_P_IP    0x0800
+#define ETH_P_IPV6  0x86DD
+
+//#include <linux/pkt_cls.h>
+#define TC_ACT_UNSPEC	(-1)
+#define TC_ACT_SHOT		2
+
+u8 rc_allow = TC_ACT_UNSPEC;
+u8 rc_disallow = TC_ACT_SHOT;
+
+struct {
+    __uint(type, BPF_MAP_TYPE_ARRAY);
+    __uint(max_entries, 10);
+    __type(value, u16);
+    __type(key, u32);
+} ports SEC(".maps");
+
+/*
+ * Determine an ip4 header
+ *
+ * @param eth, ethernet header
+ * @param data_end, end of the entire packet
+ *
+ * @return a struct iphdr if this is an ipv4 packet
+ *         NULL otherwise
+ */
+/*static struct iphdr* is_ipv4(struct ethhdr *eth, void *data_end) */
+/*{*/
+    /*struct iphdr *iph = NULL;*/
+    /*if (!eth || !data_end) {*/
+        /*return NULL;*/
```

Comment:
did you mean to remove the commented out code from this commit?

Reply:
Yes.  Sorry.  will address.

Reply:
Resolved